agent/pkg/routes/service_map_routes.go

```diff
@@ -0,0 +1,17 @@
+package routes
+
+import (
+	"mizuserver/pkg/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func ServiceMapRoutes(ginApp *gin.Engine) {
+	routeGroup := ginApp.Group("/servicemap")
+
```

Comment:
```suggestion
routeGroup.Use(middlewares.RequiresAuth())
```

Reply:
Do I need to handle auth on the front-end request as well? add any auth header?